### PR TITLE
Add cargo-rdme to supported tools

### PR DESCRIPTION
**Pull request by joshka**

### CODE REVIEW

**Comment by joshka** [comment]
The change in https://github.com/taiki-e/install-action/pull/265/commits/6b8955b3299acaef59f0307929e4d6eb7947375c uses the release zip files instead of the zst files (which CI seems to choke on)

**Comment by taiki-e** [comment]
> zst files

To use tar.zst, you have to add `*.tar.zst` branch to [this `case` block](https://github.com/taiki-e/install-action/blob/0e17f1a03533b48d71f4c916c50cf16ab2f3cd33/main.sh#L72).

**Comment by joshka** [comment]
> > zst files
> 
> To use tar.zst, you have to add `*.tar.zst` case to [this `case` block](https://github.com/taiki-e/install-action/blob/0e17f1a03533b48d71f4c916c50cf16ab2f3cd33/main.sh#L72).

I noticed that - figured I'd try to avoid it instead, not sure why the bin location was broken in the previous CI run, but figure it's better to just add zstd instead.

File: main.sh [hunk 1]
```diff
@@ -100,6 +100,18 @@ download_and_extract() {
                 esac
             fi
             ;;
+        *.tar.zst | *.tzst)
+            tar_args+=("xvf" "--zstd")
```
Comment (taiki-e) [inline main.sh:104]:
```suggestion
            tar_args+=("--zstd" "xvf")
```

(Argument after f (-f) is considered filename.)

Comment (joshka) [reply main.sh:104]: :facepalm: of course :D

**Comment by joshka** [comment]
Whew - got some green now at least :D Still failing a few OSes though.

**Comment by joshka** [comment]
Removed the zstd code as it was incompatible with macos-11, alpine, almalinux, centos (using bz2 and zip files from cargo-rdme instead). I'm sure there's a way to get that going, but my goal was not to make zstd work for install-action, it was to just get cargo-rdme available as a tool for CI.

Last change was fixing the tidy dictionary for rdme.

**Comment by joshka** [comment]
Finally all the checks are passing on this (I love when CI servers make me feel like the biggest noob :)) I've rebased the PR on main in the last push to make it easy to merge.

**Review by taiki-e — APPROVED** [review]
Thanks!

**Comment by taiki-e** [comment]
Published in 2.21.0.

**Pull request merged**